src/main/resources/errors.yaml

```diff
@@ -870,14 +870,36 @@ request-errors:
       
       Resend the command using supported model.
 
+  - scope: SCHEMA
+    code: DEPRECATED_PROVIDER_MODEL
+    title: The model is DEPRECATED.
+    body: |-
+      The model ${model} is at ${modelStatus} status.
+      ${message}
+      
+      DEPRECATED model cannot be used for creation of new Collections or Tables, but can be used for existing Collections or Tables until model status is changed to END_OF_LIFE.
+      
+      Resend the command using supported model.
+
+  - scope: SCHEMA
+    code: END_OF_LIFE_PROVIDER_MODEL
+    title: The model is ENF_OF_LIFE.
+    body: |-
+      The model ${model} is at ${modelStatus} status.
```

Comment:
The command attempted to use an AI Model that has been marked as end of life. End of life models cannot be used in any way. Collections or Tables that use the model must be recreated as data such as embeddings is not transferrable.

The model is: ${model}
${message}

Resend the command using supported model.